installers-conda/resources/base_env.yml

```diff
@@ -1,10 +1,10 @@
 channels:
   - conda-forge
 dependencies:
-  - python=3.11.9
-  - conda=24.5.0
-  - menuinst=2.1.2
-  - mamba=1.5.8  # Remove for Spyder 7
+  - python =3.12.11
+  - conda =25.7.0
```

Comment:
I think we should pin this version to `25.1.1` to work around issue #24132.

That way cached kernels would work on Windows, which means faster kernels restarts there.

Reply:
So I gave this a check on Windows with different version of conda. With conda `25.5.1`, `25.5.0`, and `25.7.0` I'm unable to reproduce the issue with the related lines commented out :+1: